scripts/package/package-debian.sh

```diff
@@ -32,12 +32,28 @@ TEST_STAGE_DIR="$PACKAGE_OUTPUT_DIR/test"
 REPO_BINARIES_DIR="$REPOROOT/artifacts/ubuntu.14.04-x64/stage2"
 MANPAGE_DIR="$REPOROOT/Documentation/manpages"
 
+NIGHTLY_PACKAGE_NAME="dotnet-nightly"
+RELEASE_PACKAGE_NAME="dotnet"
+
 execute_build(){
+    determine_package_name
     create_empty_debian_layout
     copy_files_to_debian_layout
     create_debian_package
 }
 
+determine_package_name(){
+    if [[ "$RELEASE_SUFFIX" == "dev" ]]; then
+        DOTNET_DEB_PACKAGE_NAME=$NIGHTLY_PACKAGE_NAME
+    elif [[ "beta rc1 rc2 rtm" =~ (^| )"$RELEASE_SUFFIX"($| ) ]]; then
```

Comment:
@piotrpMSFT @Sridhar-MS Are there any more identifiers we may want to consider here?